daemon/start.go

```diff
@@ -224,6 +224,17 @@ func (daemon *Daemon) containerStart(ctx context.Context, daemonCfg *configStore
 		}
 	}()
 
+	// Finish sandbox initialisation (if there's a sandbox, so not using container networking).
```

Comment:
Existence of a sandbox implies container networking, so the comment makes no sense as written. Did you mean to write this?
```suggestion
	// Finish sandbox initialisation (if there's a sandbox, so not using host networking).
```

Reply:
I do mean container networking ... in the comment I'm referring to the Sandbox object rather than the sandbox concept, perhaps that's caused the confusion - so I'll change the comment to "if there's a libnetwork.Sandbox ...", to make it less ambiguous.

But, the line of code it's commenting on calls `daemon.netController.GetSandbox(container.ID)` ... for host networking, that retrieves the host network's Sandbox. For container networking it returns nil, because there's no new Sandbox associated with the new container.

Reply:
The "mode" originated from before we had custom networks. At that time, the `--net` option was `bridge`, `none`, or `host` (possibly we already had `container` as well).

When custom networks were added, the `--net` / `--network` option was expanded to specify either (existing `bridge`, `none`, and `host` options, as well as `<name of custom network>`)